### PR TITLE
docs: matomo perf tunning again

### PR DESCRIPTION
**Pull request by alexgarel**

### CODE REVIEW

**Review by stephanegigandet — APPROVED** [review]
Thanks, hopefully it will improve things :)

File: docs/matomo.md [hunk 1]
```diff
@@ -13,13 +13,21 @@ Ask for it to *contact* email.
...
 
 Important configuration files are linked to this git repository (see [using git, below](#using-git))
 
+## Alerts
+
+You can setup alerts in matomo. In personal alerts.
+
+We have setup alerts for main website and app if there are 0 visits.
+
+![alerts settings](./img/matomo-alerts.png "Matomo aletrs settings")
```
Comment (CharlesNepote) [inline docs/matomo.md:29]:
```suggestion
![alerts settings](./img/matomo-alerts.png "Matomo alerts settings")
```

File: docs/reports/2024-01-matomo-perf-tunning [hunk 2]
```diff
@@ -0,0 +1,230 @@
+# 2024-01 Matomo performanc tunning
```
Comment (CharlesNepote) [inline docs/reports/2024-01-matomo-perf-tunning:1]:
```suggestion
# 2024-01 Matomo performance tunning
```

**Review by CharlesNepote — APPROVED** [review]
Great!

**Pull request merged**